tools/python/setup.py

```diff
@@ -10,7 +10,8 @@
          "termcolor",
          "pyyaml",
          "jsonschema",
-         "tabulate"
+         "tabulate",
+         "mock"
```

Comment:
mock seems to be part of the Python library from 3.3. Not sure if it changes anything here.

Reply:
It is, but it seems that everything still works fine when it's manually installed.